libminifi/include/utils/FlatMap.h

```diff
@@ -138,7 +134,9 @@ class FlatMap{
     return data_.size();
   }
 
-  V& operator[](const K& key) {
+  template<typename T>
+  requires std::constructible_from<K, T> && std::equality_comparable_with<K, T>
```

Comment:
Why do we need `constuctible_from`? If we really need it, are we really interested in constructing K from an object of T?

Reply:
operator[] insert a default constructred value at the specified key if its not already there. for that we need to create the actual keytype thats why we need the constraint